sparse_strips/vello_common/src/flatten_simd.rs

```diff
@@ -111,17 +112,12 @@ pub(crate) fn flatten<S: Simd>(
                         callback.callback(PathEl::LineTo(p3));
                     } else {
                         let c = CubicBez::new(p0, p1, p2, p3);
-                        let max = simd.vectorize(
-                            #[inline(always)]
-                            || {
-                                flatten_cubic_simd(
-                                    simd,
-                                    c,
-                                    flatten_ctx,
-                                    tolerance as f32,
-                                    &mut flattened_cubics,
-                                )
-                            },
+                        let max = flatten_cubic_simd(
+                            simd,
+                            c,
+                            flatten_ctx,
+                            tolerance as f32,
+                            &mut flattened_cubics,
```

Comment:
This `vectorize` is no longer necessary, as the `flatten_cubic_simd` call gets inlined into `flatten` which itself gets vectorized.